CMakeLists.txt

```diff
@@ -174,17 +176,14 @@ if(NOT ONNX_BUILD_CUSTOM_PROTOBUF)
                   NAMES protoc
                   PATHS ${_PROTOBUF_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}
                   NO_DEFAULT_PATH)
-      if(ONNX_USE_LITE_PROTO)
-        find_library(Protobuf_LITE_LIBRARY
-          NAMES protobuf-lite
-          PATHS ${_PROTOBUF_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}
-          NO_DEFAULT_PATH)
-      else()
-        find_library(Protobuf_LIBRARY
-          NAMES protobuf
-          PATHS ${_PROTOBUF_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}
-          NO_DEFAULT_PATH)
-      endif(ONNX_USE_LITE_PROTO)
+      find_library(Protobuf_LITE_LIBRARY
```

Comment:
Why not keeping the test? This we don't have to look for a library we don't need.

Reply:
The find_library calls are used to provide variables for the following find_package(Protobuf). I thought it's better to give as more variables as possible. Another consideration is that even though ONNX_USE_LITE_PROTO is set, it's possible to fall back to protobuf if protobuf-lite doesn't exist.

Reply:
I assume there is no risk one step fails because it is missing.

Reply:
Alright, I revert to the old code.